### ISSUE TITLE
Savage Attack doesn't work on sneak attack dice

### ISSUE THREAD

**Issue by ChrisPJohn** [ChrisPJohn/SolastaCommunityExpansion#169]
ndo317
I can confirm that sneak dices are not rerolled. Is this intended ? How about other dices such as spell damage dices or smite damage dices ?

--------------------------------

Most likely this means it won't work on smite dice.

If this is intended behavior the text should be updated to represent that.

**Comment by ThyWoof** [comment]
The Weapon's damage dice is different from the total damage dice.

Savage Attacker works with allowing reroll on the weapon's damage dice.

Sneak attack and it's associated dice are a different mechanic and represent extra damage on top of the Weapon dice: it is not part of the weapon dice.

https://twitter.com/JeremyECrawford/status/679830252005101568

**Comment by ChrisPJohn** [comment]
The feat in CE works on spell damage, so it isn't identical to the table top feat. This was done partially because at the time it was implemented weapon and spell damage rolls were still on a single context for additional damage. Though I did like giving the feat a little boost to usefulness.

I'm not saying we need to change it so the CE Savage Attacker feat works on sneak attack damage, but as I mentioned in the original message, if we aren't changing it then we should update the text for the feat because the text currently says: "Reroll damage dice when they roll a 1." Which can easily be interpreted to include things like sneak attack.
